homeassistant/components/fronius/__init__.py

```diff
@@ -144,6 +150,25 @@ async def init_devices(self) -> None:
             )
         )
 
+    async def _create_solar_net_device(self) -> None:
+        """Create a device for the Fronius SolarNet system."""
+        solar_net_device: DeviceInfo = DeviceInfo(
+            configuration_url=self.host,
+            name="SolarNet",
+            identifiers={(DOMAIN, self.solar_net_device_id)},
+            manufacturer="Fronius",
+        )
+        if self.logger_coordinator:
+            _logger_info = self.logger_coordinator.data[SOLAR_NET_ID_SYSTEM]
+            solar_net_device["model"] = _logger_info["product_type"]["value"]
+            solar_net_device["sw_version"] = _logger_info["software_version"]["value"]
```

Comment:
Since solar_net_device is a DeviceInfo object, you can use the ATTR_MODEL and ATTR_SW_VERSION constants

Reply:
Thank you very much! 👍